realm/realm-library/src/main/cpp/io_realm_internal_Property.cpp

```diff
@@ -63,7 +63,7 @@ JNIEXPORT jlong JNICALL Java_io_realm_Property_nativeCreateProperty__Ljava_lang_
     return 0;
 }
 
-JNIEXPORT void JNICALL Java_io_realm_Property_nativeClose(JNIEnv* env, jclass, jlong property_ptr)
+JNIEXPORT void JNICALL Java_io_realm_internal_Property_nativeClose(JNIEnv* env, jclass, jlong property_ptr)
```

Comment:
That is more like for the future, the `RealmObject` will hold a `OsObjectSchemaInfo` because of lots of function calls from Object Store take a `ObjectSchema` as an input.
Also, Proxy hold a static `OsObjectSchemaInfo` which in theory doesn't have to be closed, but phantom ref makes it literally correct.
BTW, by following the phantom ref pattern makes our memory management life much easier.